src/algebra/group/units.lean

```diff
@@ -117,16 +125,12 @@ lemma copy_eq (u : αˣ) (val hv inv hi) :
 ext hv
 
 /-- Units of a monoid form a group. -/
-@[to_additive "Additive units of an additive monoid form an additive group."] instance : group αˣ :=
-{ mul := λ u₁ u₂, ⟨u₁.val * u₂.val, u₂.inv * u₁.inv,
-    by rw [mul_assoc, ← mul_assoc u₂.val, val_inv, one_mul, val_inv],
-    by rw [mul_assoc, ← mul_assoc u₁.inv, inv_val, one_mul, inv_val]⟩,
-  one := ⟨1, 1, one_mul 1, one_mul 1⟩,
-  mul_one := λ u, ext $ mul_one u,
-  one_mul := λ u, ext $ one_mul u,
-  mul_assoc := λ u₁ u₂ u₃, ext $ mul_assoc u₁ u₂ u₃,
+@[to_additive "Additive units of an additive monoid form an additive group."]
+instance : group αˣ :=
+{ mul_assoc := λ u₁ u₂ u₃, ext $ mul_assoc u₁ u₂ u₃,
```

Comment:
IIRC, specifying `mul` means the unfolded version of `units.group` will give theorems stated using `*` instead of `mul_one_class.to_has_mul.mul` or something like that.